posix/posix.c

```diff
@@ -1576,22 +1553,21 @@ int posix_fcntl(int fd, unsigned int cmd, char *ustack)
 #define SIOCDELRT   _IOC(IOC_IN, 'S', 0x45, sizeof(struct rtentry))
 
 
-void ioctl_pack(msg_t *msg, unsigned long request, void *data, id_t id)
+static void ioctl_pack(msg_t *msg, unsigned long request, void *data, oid_t oid)
```

Comment:
Might be better to pass via pointer, some performance penalty in passing struct by value